lmnet/configs/core/segmentation/lm_segnet_v1_quantize_camvid.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 The Blueoil Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+from easydict import EasyDict
+import tensorflow as tf
+
+from lmnet.common import Tasks
+from lmnet.networks.segmentation.lm_segnet_v1 import LmSegnetV1Quantize
+from lmnet.datasets.camvid import Camvid
+from lmnet.data_processor import Sequence
+from lmnet.pre_processor import (
+    DivideBy255,
+)
+from lmnet.data_augmentor import (
+    Brightness,
+    Color,
+    Contrast,
+    FlipLeftRight,
+    Hue,
+)
+from lmnet.quantizations import (
+    binary_mean_scaling_quantizer,
+    linear_mid_tread_half_quantizer,
+)
+
+IS_DEBUG = False
+
+NETWORK_CLASS = LmSegnetV1Quantize
+DATASET_CLASS = Camvid
+
+IMAGE_SIZE = [360, 480]
+BATCH_SIZE = 8
+DATA_FORMAT = "NHWC"
+TASK = Tasks.SEMANTIC_SEGMENTATION
+CLASSES = DATASET_CLASS.classes
+
+MAX_STEPS = 150000
+SAVE_STEPS = 3000
+TEST_STEPS = 1000
+SUMMARISE_STEPS = 1000
+
+# distributed training
+IS_DISTRIBUTION = False
+
+# pretrain
+IS_PRETRAIN = False
+PRETRAIN_VARS = []
+PRETRAIN_DIR = ""
+PRETRAIN_FILE = ""
+
+# for debug
+# BATCH_SIZE = 2
+# SUMMARISE_STEPS = 1
+# IS_DEBUG = True
+
+PRE_PROCESSOR = Sequence([
+    DivideBy255(),
```

Comment:
@nlpng 
For inference, if the input image size is not  `[360, 480]`, It is good to set `Resize(size=IMAGE_SIZE),`.

Reply:
Thanks, done adding the resize preprocessor in https://github.com/blue-oil/blueoil/pull/42/commits/ca4ae9788ffa65e148d9201af5d35fa84170dc2a.